swagger/apidocs/specs/openapi.yaml

```diff
@@ -0,0 +1,88 @@
+swagger: '2.0'
```

Comment:
If we merge this will we be exposing private endpoints?

Reply:
True!